tests/unit/src/main/scala/tests/codeactions/BaseCodeActionLspSuite.scala

```diff
@@ -33,7 +33,8 @@ abstract class BaseCodeActionLspSuite(suiteName: String)
       scalafixConf: String = "",
       scalacOptions: List[String] = Nil,
       configuration: => Option[String] = None,
-      scalaVersion: String = scalaVersion
+      scalaVersion: String = scalaVersion,
+      extraOperations: Option[() => Unit] = None
```

Comment:
Why is this needed?

Reply:
I didn't want to create a new `check` function with duplicated code just to add extra checks at the end in the same workspace, so I added `extraOperations` that are used as following:
```
extraOperations = Some(() => {
        val (fileName, content) = newFile
        val absolutePath = workspace.resolve(getPath(fileName))
        assert(
          absolutePath.exists,
          s"File $absolutePath should have been created"
        )
        assertNoDiff(absolutePath.readText, content)
      })
      ```

Reply:
Its type might be simplified to
```scala
extraOperations: => Unit = ()
```